src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements AreaCalculetion {
+    private double sideB;
+    private double sideA;
+
+    public Rectangle(double sideA, double sideB, String color, String name) {

```

Comment:
The constructor should not omit setting the color and name for the `Rectangle`. These properties should be passed to the superclass `Figure` constructor using `super(color, name);`.